<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRoom</class>
 <widget class="QWidget" name="ChatRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="exitRoomBt">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>570</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="messageBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>591</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sendEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>591</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>发送消息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendFileBt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>570</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
  </widget>
  <widget class="QListView" name="usersList">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>161</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="filesList">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>300</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="userLb">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>在线用户</string>
   </property>
  </widget>
  <widget class="QLabel" name="fileLb">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>270</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>文件列表</string>
   </property>
  </widget>
  <widget class="QLabel" name="fileSendLb">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>460</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>文件传输</string>
   </property>
  </widget>
  <widget class="QLabel" name="userNumLb">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QListView" name="filessendList">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>490</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
